controllers/policyset/policyset_controller.go

```diff
@@ -203,11 +204,17 @@ func (r *PolicySetReconciler) processPolicySet(ctx context.Context, plcSet *poli
 			// aggregate compliance state
 			plcComplianceState := complianceInRelevantClusters(childPlc.Status.Status, clusters)
 			if plcComplianceState == "" {
-				pendingPolicies = append(pendingPolicies, string(childPlcName))
+				unknownPolicies = append(unknownPolicies, string(childPlcName))
 			} else {
-				compliancesFound = append(compliancesFound, string(childPlcName))
-				if plcComplianceState == "NonCompliant" {
-					aggregatedCompliance = "NonCompliant"
+				if plcComplianceState == policyv1.Pending &&
+					aggregatedCompliance != policyv1.NonCompliant {
+					pendingPolicies = append(pendingPolicies, string(childPlcName))
```

Comment:
What is the reason for not appending to `pendingPolicies` when `aggregatedCompliance == policyv1.NonCompliant`? I think we only want to not set `aggregatedCompliance` in this case.